### PR TITLE
fixes for Reupen's comments

### PR DESCRIPTION
**Pull request by jim68000**
I need to come up with a better way of handling generated files. Problem here is that I need to generate a separate node application to run the stubs, and that confuses heroku.

### CODE REVIEW

File: gendocs/api/swagger.yaml [hunk 1]
```diff
@@ -242,12 +242,11 @@ paths:
         400:
```
Comment (jim68000) [inline gendocs/api/swagger.yaml]: generated file, can be ignored

File: gendocs/controllers/CompanyService.js [hunk 2]
```diff
@@ -74,8 +74,9 @@ exports.getcompany = function(args, res, next) {
   "archived" : true,
```
Comment (jim68000) [inline gendocs/controllers/CompanyService.js]: Generated file, can be ignored

File: gendocs/controllers/InvestmentService.js [hunk 3]
```diff
@@ -33,71 +33,69 @@ exports.getinvestment = function(args, res, next) {
...
-      "id" : "d290f1ee-6c54-4b01-90e6-d701748f0851"
-    },
-    "investment_recipient_company" : "",
-    "non_fdi_type" : {
-      "name" : "Advanced Engineering Supply Chain",
-      "id" : "d290f1ee-6c54-4b01-90e6-d701748f0851"
-    },
-    "name" : "Marriot Hotels - New hotel in Manchester",
-    "intermediate_company" : ""
+  "estimated_land_date" : "2017-05-01",
```
Comment (jim68000) [inline gendocs/controllers/InvestmentService.js]: Generated file, can be ignored

File: public/index.html [hunk 4]
```diff
@@ -13,7 +13,7 @@ <h1 id="data-hub-crm-api-spec">Data Hub CRM API Spec</h1>
 </ul>
```
Comment (jim68000) [inline public/index.html]: Generated file, can be ignored

File: generator/specs/definitions/company_defs.yml [hunk 5]
```diff
@@ -127,15 +127,19 @@
...
-    - name
+    - firstname
+    - lastname
     properties:
       id:
         type: string
         format: uuid
         example: 'd290f1ee-6c54-4b01-90e6-d701748f0851'
-      name:
+      firstname:
```
Comment (reupen) [inline generator/specs/definitions/company_defs.yml]: Should be `first_name` and `last_name` for consistency with Contact (and model)?

Comment (jim68000) [inline generator/specs/definitions/company_defs.yml]: Good point. Done.

**Pull request merged**